src/components/Questionnaire/QuestionTypes/TemplateSelector.tsx

```
@@ -0,0 +1,107 @@
+          {templates.map((template) => (
+            <div
+              key={template.name}
+              className="flex items-center space-x-3 p-2 hover:bg-gray-100 rounded cursor-pointer"
+              onClick={() => {
+                setSelectedTemplates((prev) =>
+                  prev.includes(template.name)
+                    ? prev.filter((t) => t !== template.name)
+                    : [...prev, template.name],
+                );
+              }}
+            >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)